tests/jobs/test_scheduler_job.py

```diff
@@ -293,11 +295,7 @@ def test_execute_task_instances_is_paused_wont_execute(self):
             max_active_tasks=dag.max_active_tasks,
             has_task_concurrency_limits=False,
         )
```

Comment:
This block here for instance where we create a DagModel appears 23 times in this file, so this should be moved in side the DagMaker